tool/site/client-react/src/components/selectField.tsx

```diff
@@ -1,11 +1,10 @@
-import React, {FC} from "react"
+import React, {FC, HTMLProps} from "react"
 import {Select} from "nhsuk-react-components"
 import {Field} from "formik"
 import {Coding} from "fhir/r4"
+import {FormElementProps} from "nhsuk-react-components/dist/util/types/FormTypes"
```

Comment:
can we import this from same place as line 2?

Reply:
No